js/gemini.js

```diff
@@ -78,8 +78,8 @@ module.exports = class gemini extends Exchange {
             },
             'fees': {
                 'trading': {
-                    'taker': 0.0025,
-                    'maker': 0.0025,
+                    'taker': 1,
```

Comment:
100% fees? o.O

Reply:
Oops.  Thanks.  This is why (I think) we should use text eg "1%", like the exchanges do.  Then just convert to a decimal at runtime.

Reply:
FYI - I'n my project, to avoid exactly this type of error I use

`Util.percentToDecimal("0.1%");`

and

`Util.decimalToPercent`(...

Reply:
@npomfret yeah, we do acknowledge that it may be hard to remember, however, it isn't really a matter of conversion, because people will still confuse it if not this way, then the other way around – someone could mistakenly write `Util.percentToDecimal("0.01%")` for gemini that would be a typo as well, so it is not directly relevant to formatting or guards, but we will add similar wrappers most likely. Thx!